src/components/AddPlaylistBox/AddPlaylistBox.css

```diff
@@ -0,0 +1,59 @@
+.add-playlist-box {
+  background-color: var(--theme-clr);
+  max-width: 30rem;
+  width: max-content;
+  flex-direction: column;
+  color: var(--clr-white);
+  z-index: 200;
+  align-items: flex-start;
+  padding: 1rem;
+  border-radius: 5px;
+}
+.add-playlist-box .close-icon {
+  width: 2.5rem;
+  height: 2.5rem;
+}
+.checkbox {
+  border-radius: 1rem;
+  height: 1.5rem;
+  width: 1.5rem;
+  outline: none;
+  margin-right: 1rem;
+}
+.playlist-list {
+  align-items: flex-start;
+  flex-direction: column;
+}
+.add-playlist-box li {
+  margin: 1rem 0;
+  list-style-type: none;
+}
+.add-playlist-box .icon-box {
+  justify-content: space-between;
+  width: 100%;
+  gap: 1rem;
+}
+.overlay-box {
+  position: fixed;
+  opacity: 1;
+  width: 100vw;
+  background: var(--overlay-clr);
+  height: 100%;
+  left: 0;
+  z-index: 200;
```

Comment:
don't provide very large z-index value

Reply:
Actually I have provided large value in my component library so need to go according to it.